guides/source/7_2_release_notes.md

```diff
@@ -36,10 +36,39 @@ Please refer to the [Changelog][railties] for detailed changes.
 
 *   Remove deprecated `find_cmd_and_exec` console helper.
 
+*   Remove support for `oracle`, `sqlserver` and `JRuby` specific database adapters from the rails new
```

Comment:
Should `JRuby` be `jruby` instead? Both commands could also be in code blocks

Reply:
Officially its referring to JRuby

Reply:
Shouldn't it be `Oracle` then as well? Or is that literally how I would prefix the db driver to use these variants?

Reply:
Actually These all the are the options values which supported by generator option `--database`. The JRuby container `sqlserver` `jdbcmysql` `jdbcsqlite3` `jdbcpostgresql` `jdbc`